docs/about.md

```diff
@@ -103,40 +103,40 @@ We also support the [NHS Data Science Community](https://data-science-community.
     <tr><td>Giulia Mantovani</td><td>Data Science Lead</td><td>Data Linkage Hub</td><td><a href="https://github.com/GiuliaMantovani1">GiuliaMantovani1</a></td> </tr>
     <tr><td>Angeliki Antonarou</td><td>Principal Data Scientist</td><td>National SDE Data Science Team</td><td><a href="https://github.com/AngelikiA">AnelikiA</a></td> </tr>
     <tr><td>Kevin Fasusi</td><td>Principal Data Scientist</td><td>National SDE Data Science Team</td><td><a href="https://github.com/KevinFasusi">KevinFasusi</a></td> </tr>
```

Comment:
looking at the org chart it looks like Kevin is gone from SDE https://nhsd-confluence.digital.nhs.uk/spaces/DAT/pages/315334104/Org+chart?preview=/315334104/930368287/DS%20org%20chart.xlsx

Reply:
also Giulia's title should be "Head of Data Linkage Hub"